index.html

```diff
@@ -0,0 +1,147 @@
+                </ul>
+          
+                <button class="see-project-btn">See Project</button>
+                
+            </div>
+            <div class="works">
+                <h3 class="works-title">Profesional Art <br>Printing Data</h3>
+                <p class="font-montesori">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tools-container">
+                    <li class="tools bg-btn">html</li>
+                    <li class="tools bg-btn">css</li>
+                    <li class="tools bg-btn">bootstrap</li>
+                    <li class="tools bg-btn">Ruby</li>
+                </ul>
+                
+                <button class="see-project">See Project</button>
+
+            </div>
+            <div class="works">
+                <h3 class="works-title">Profesional Art <br>Printing Data</h3>
+                <p class="font-montesori">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tools-container">
+                    <li class="tools bg-btn">html</li>
+                    <li class="tools bg-btn">css</li>
+                    <li class="tools bg-btn">bootstrap</li>
+                    <li class="tools bg-btn">Ruby</li>
+                </ul>
+                
+                <button class="see-project">See Project</button>
+
+            </div>
+            <div class="works">
+                <h3 class="works-title">Profesional Art <br>Printing Data</h3>
+                <p class="font-montesori">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tools-container">
+                    <li class="tools bg-btn">html</li>
+                    <li class="tools bg-btn">css</li>
+                    <li class="tools bg-btn">bootstrap</li>
+                    <li class="tools bg-btn">Ruby</li>
+                </ul>
+                
+                <button class="see-project">See Project</button>
+
+            </div>
+           
+
+        </section>
+        <section id="about-me-container">
+
+            <div class="about-me-block">
+                <h3 class="works-title yelow abt-me">About me</h3>
+                <p class="font-montesori  white">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button class="see-project-btn">Get My Resume</button>
```

Comment:
- [ ] Kindly fix this button to match the design template.
### Your design
![IMG_20221202_181800](https://user-images.githubusercontent.com/94551484/205348948-ac11452b-1c17-4a52-913a-9a95e7d1f111.jpg)
### Figma template
![IMG_20221202_181721](https://user-images.githubusercontent.com/94551484/205349035-9b9971f3-e835-41e6-9be0-1b095792529a.jpg)